web/src/components/HeaderNavigationItem.tsx

```diff
@@ -129,7 +129,7 @@ export type HeaderNavigationItemProps = {
 }
 
 const HeaderNavigationItem = ({ active, text, href, icon, direction }: HeaderNavigationItemProps): ReactElement => (
-  <Container direction={direction}>
+  <Container direction={direction} className='navigation-item'>
```

Comment:
Optional: Would prefer 
```suggestion
  <Container direction={direction} className='header-navigation-item'>
```

Reply:
fixed in last commit